dev/package-examples/nginx-1.2.0/dataset/access/manifest.yml

```diff
@@ -8,18 +8,23 @@ type: logs
 # The ingest pipeline which should be used.
 ingest_pipeline: default
 
-vars:
-  - name: paths
-    # Should we define this as array? How will the UI best make sense of it?
-    type: textarea
-    default:
-      - /var/log/nginx/access.log*
-    # I suggest to use ECS fields for this config options here: https://github.com/elastic/ecs/blob/master/schemas/os.yml
-    # This would need to be based on a predefined definition on what can be filtered on
-    os.darwin:
-      - /usr/local/var/log/nginx/access.log*
-    os.windows:
-      - c:/programdata/nginx/logs/*access.log*
+# List of supported inputs
+inputs:
+  - type: log
```

Comment:
@hbharding suggested today that it would be great to have descriptions in the UI. Could we add `description` prop for every input, under its `type`? I think these descriptions should only be a sentence or two, max. Not sure how we will localize these though..